include/clap/helpers/host.hh

```diff
@@ -51,6 +52,7 @@ namespace clap { namespace helpers {
       virtual void requestCallback() noexcept = 0;
 
       virtual bool enableDraftExtensions() const noexcept { return false; }
+      virtual const void* getExtension (std::string_view extensionId) const noexcept { return nullptr; }
```

Comment:
```suggestion
      virtual const void *extension(const char *id) const noexcept { return nullptr; }
```
- rename for consistency with `clap::helpers::Plugin::extension`
- just pass on the `const char *` -> let it up to the implementation to use `std::string_view`
- format: ` void *extension(` instead of ` void* extension (` seems more consistent with the rest to me

Reply:
> rename for consistency with clap::helpers::Plugin::extension

`Plugin` uses `clapExtension` and `extension`, but `Host` uses `clapGetExtension` so the corresponding method would be `getExtension`. However, the parameter type and name should be changed to `const char *extension_id` to match `Host::clapGetExtension`.

Reply:
Should I change it back... probably should have looked here first. :)

Reply:
Yeah, I actually think messmerd is more right than me :D
So maybe it would be ok for you to change the name to `getExtension`...?